api/client.go

```diff
@@ -123,11 +123,23 @@ func NewClient(account string, opts ...Option) (*Client, error) {
 	return c, nil
 }
 
-// WithSubaccount sets a subaccount
+// RemoveSubaccount removes the subaccount from the API client
+func (c *Client) RemoveSubaccount() {
```

Comment:
What situation would the subaccount need to be removed?

^ Not a comment on the usefulness of this function, genuinely not sure of the use case to remove the subaccount.

Reply:
To access the primary account, instead of the sub-account

Reply:
Just realized that this was part of the next change for the migration. I'll remove it from here. 👍🏽